src/graph/graph.c

```
@@ -1408,6 +1408,105 @@ static void _Graph_Free
+
+	//--------------------------------------------------------------------------
+	// Graph's relation matrices
+	//--------------------------------------------------------------------------
+
+	int n_rel = Graph_RelationTypeCount(g);
+	for(RelationID rel = 0; rel < n_rel; rel++) {
+		T = Graph_GetRelationMatrix(g, rel, false);
+		M = Delta_Matrix_M(T);
+
+		info = GxB_Matrix_memoryUsage(&n, M);
+		ASSERT(info == GrB_SUCCESS);
```

Cppcheck (2.10-2)

[error] Null pointer dereference

(nullPointer)